examples/text_processing/g2p/heteronym_classification_inference.py

```diff
@@ -33,21 +33,39 @@
 
 To prepare dataset, see NeMo/scripts/dataset_processing/g2p/export_wikihomograph_data_to_manifest.py
 
+Inference form manifest:
+
 python heteronym_classification_inference.py \
     manifest="<Path to .json manifest>" \
     pretrained_model="<Path to .nemo file or pretrained model name from list_available_models()>" \
-    output_file="<Path to .json manifest to save prediction>"
+    output_manifest="<Path to .json manifest to save prediction>" \
+    wordid_to_phonemes_file="<Path to a file with mapping from wordid predicted by the model to phonemes>"
+
+Interactive inference:
+
+python heteronym_classification_inference.py \
+    pretrained_model="<Path to .nemo file or pretrained model name from list_available_models()>" \
+    wordid_to_phonemes_file="<Path to a file with mapping from wordid predicted by the model to phonemes>" # Optional
+        
 """
 
 
 @dataclass
 class TranscriptionConfig:
     # Required configs
     pretrained_model: str  # Path to a .nemo file or Name of a pretrained model
-    output_file: str  # Path to .json manifest to save prediction, will be saved in "pred_text" field
-    manifest: str  # Path to .json manifest
+
+    # path to .json manifest inference, if not provided or doesn't exist, interactive mode will be enabled
```

Comment:
Can get rid of "or doesn't exist" here, right?

Reply:
yep, removed